cli/src/main.rs

```diff
@@ -252,6 +252,15 @@ fn run() -> Result<(), CliError> {
                                 .long("member")
                                 .help("Filter the circuits by a node ID in the member list")
                                 .takes_value(true),
+                        )
+                        .arg(
+                            Arg::with_name("format")
+                                .short("f")
+                                .long("format")
+                                .help("Format of the circuit")
+                                .possible_values(&["human-readable", "csv"])
```

Comment:
Suggest we change 'human-readable' to 'default'. (As in, the "default" is human readable.)